frontend/public/components/utils/dropdown.jsx

```diff
@@ -289,15 +289,18 @@ export class Dropdown extends DropdownMixin {
     const {active, autocompleteText, selectedKey, items, title, bookmarks, keyboardHoverKey, favoriteKey} = this.state;
     const {autocompleteFilter, autocompletePlaceholder, noButton, noSelection, className, menuClassName, storageKey, canFavorite, dropDownClassName} = this.props;
 
-    const button = noButton
-      ? <div onClick={this.toggle} className="dropdown__not-btn" id={this.props.id}>
+    let button;
+    if (noButton) {
+      button = <div onClick={this.toggle} className="dropdown__not-btn" id={this.props.id}>
```

Comment:
Ideally this would be a button element anyway for accessibility, just without the button styling. Not asking you to fix that in this PR... Just noting the existing dropdown is not keyboard or screen reader accessible.

We should open a JIRA issue.